SPECS/protobuf/protobuf.spec

```diff
@@ -53,24 +71,69 @@ Provides:       %{name}-python3 = %{version}-%{release}
 %description -n python3-%{name}
 This contains protobuf python3 libraries.
 
+%package -n     java-%{name}
```

Comment:
Do we really want -n here?  Kinda feels like this should be Protobuf-Java

Reply:
Changed it to protobuf-java